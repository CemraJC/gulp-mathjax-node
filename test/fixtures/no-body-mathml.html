<head>
    <meta charset="utf-8">
    <title>No Body Tag</title>
</head>
<!-- In a case where there's no body tag, who cares what is going on, we just want it to work without truncating any html... -->

<script type="math/tex">e^{i\tau}=1</script>
$e^{i\pi}=-1$

<script type="math/tex; mode=display">\int_0^6 \frac{5x}{3} dx</script>
$$
    \int_0^6 \frac{5ex}{3} dex
$$