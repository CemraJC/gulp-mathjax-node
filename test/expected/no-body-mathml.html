
    <meta charset="utf-8">
    <title>No Body Tag</title>

<!-- In a case where there's no body tag, who cares what is going on, we just want it to work without truncating any html... -->

<math xmlns="http://www.w3.org/1998/Math/MathML"><msup><mi>e</mi><mrow class="MJX-TeXAtom-ORD"><mi>i</mi><mi mathvariant="italic">τ</mi></mrow></msup><mo>=</mo><mn>1</mn></math>
<math xmlns="http://www.w3.org/1998/Math/MathML"><msup><mi>e</mi><mrow class="MJX-TeXAtom-ORD"><mi>i</mi><mi mathvariant="italic">π</mi></mrow></msup><mo>=</mo><mo>−</mo><mn>1</mn></math>

<math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><msubsup><mo stretchy="false">∫</mo><mn>0</mn><mn>6</mn></msubsup><mfrac><mrow><mn>5</mn><mi>x</mi></mrow><mn>3</mn></mfrac><mi>d</mi><mi>x</mi></math>
<math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><msubsup><mo stretchy="false">∫</mo><mn>0</mn><mn>6</mn></msubsup><mfrac><mrow><mn>5</mn><mi>e</mi><mi>x</mi></mrow><mn>3</mn></mfrac><mi>d</mi><mi>e</mi><mi>x</mi></math>